lib/resources/example_json/r4_examples_person.rb

```diff
@@ -0,0 +1,323 @@
+                            "userSelected": false
+                        }
+                    ],
+                    "text": "MRN"
+                },
+                "value": "10000086",
+                "period": {
+                    "start": "2006-09-20T22:28:53.000Z"
+                }
+            }
+        ],
+        "name": [
+            {
+                "id": "CI-725944-0",
+                "use": "official",
+                "text": "SMITH, MORGAN",
+                "family": "SMITH",
+                "given": [
+                    "MORGAN"
+                ],
+                "period": {
+                    "start": "2006-09-20T21:55:51.000Z"
+                }
+            },
+            {
+                "id": "CI-17446991-0",
+                "use": "official",
+                "text": "lastName Jr., firstName middleName",
+                "family": "lastName",
+                "given": [
+                    "firstName",
+                    "middleName"
+                ],
+                "prefix": [
+                    "Dr."
+                ],
+                "suffix": [
+                    "Jr."
+                ],
+                "period": {
+                    "start": "2014-01-20T15:48:39.000Z",
+                    "end": "2014-03-20T15:48:39.000Z"
+                }
+            }
+        ],
+        "telecom": [
+            {
+                "id": "CI-PH-693923-0",
+                "system": "phone",
+                "value": "8168675234",
+                "use": "home",
+                "rank": "1",
+                "period": {
+                    "start": "2006-09-20T21:55:51.000Z"
+                }
+            }
+        ],
+        "gender": "female",
+        "birthDate": "1970-01-01",
+        "address": [
+            {
+                "id": "CI-717915-0",
+                "use": "home",
+                "text": "123 Sesame Street\nKansas City, MO 64112\nUSA",
+                "line": [
+                    "123 Sesame Street"
+                ],
+                "city": "Kansas City",
+                "district": "Jackson",
+                "state": "MO",
+                "postalCode": "64112",
+                "country": "USA",
+                "period": {
+                    "start": "2006-09-20T21:55:51.000Z"
+                }
+            }
+        ],
+        "active": true
+        }
+  
+      R4_PERSON_BUNDLE ||= {
+        "resourceType": "Bundle",
+        "id": "d44e3941-2944-455f-b3d0-a34d3039e470",
+        "type": "searchset",
+        "link": [
+            {
+                "relation": "self",
+                "url": "https://fhir-open.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Person?identifier=urn%3Aoid%3A1.1.1.1.1.1%7C10000360"
+            }
+        ],
+        "entry": [
+            {
+                "fullUrl": "https://fhir-open.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Person/1316024",
+                "resource": {
```

Comment:
Helps reduce the redundancy: https://github.com/cerner/fhir.cerner.com/blob/master/lib/resources/example_json/r4_examples_allergy_intolerance.rb#L135

Reply:
Added in commit 6b67bcc19cfed472c7ec020f38e7d45ab0067dcc